librz/include/rz_analysis.h

```diff
@@ -453,6 +446,17 @@ typedef struct rz_analysis_hint_cb_t {
 
 typedef struct rz_analysis_il_vm_t RzAnalysisILVM;
 
+typedef struct {
+	HtUP /*<ut64, RzAnalysisDwarfFunction *>*/ *function_by_offset;
+	HtUP /*<ut64, const RzAnalysisDwarfFunction *>*/ *function_by_addr;
+	HtUP /*<ut64, RzCallable *>*/ *callable_by_offset;
+	HtUP /*<ut64, RzType *>*/ *type_by_offset;
+	HtUP /*<ut64, const RzBaseType *>*/ *base_type_by_offset;
+	RzBinDwarfEncoding encoding;
+	DWARF_RegisterMapping dwarf_register_mapping;
+	RzBinDwarf *dw;
+} RzAnalysisDebugInfo;
```

Comment:
Please document this. Also the structs below. Since it is pretty central to the whole module, it should have proper documentation (e.g. what index number the hash tables is etc.).